src/domain/documentRequestFactory.ts

```diff
@@ -4,12 +4,13 @@ import getCertificate from './getCertificate';
 import getLetter from './getLetter';
 import getPlate from './getPlate';
 import getFile from './getFile';
+import getZip from './getZip';
 
 const {
   NODE_ENV, BUCKET, BRANCH,
 } = process.env;
 
-export default async (vin: string, testNumber: string, plateSerialNumber: string, systemNumber: string, fileName: string): Promise<APIGatewayProxyResult> => {
+export default async (vin: string, testNumber: string, plateSerialNumber: string, systemNumber: string, fileName: string, adrDocumentId: string): Promise<APIGatewayProxyResult> => {
```

Comment:
will it never come in an undefined'?

Reply:
It will only go through the isZip flow if it is defined, identical logic to the 4 other query parameters.